lib/activemodel_email_address_validator/email_address.rb

```diff
@@ -34,6 +34,7 @@ def valid_using_regex?(regex)
 
     def valid_using_default?
       return false if /(\s|["'<>,])+/.match?(address)
+      return false unless URI::MailTo::EMAIL_REGEXP.match?(address)
```

Comment:
I wonder if we could remove the regex-check at line 36 without any regressions. I'd expect `URI::MailtTo::EMAIL_REGEXP` to handle those cases as well.